__tests__/components/Comments/Comment.test.js

```diff
@@ -77,6 +78,17 @@ describe('Comment test should run correctly', () => {
     'https://res.cloudinary.com/hpjivutj2/image/upload/v1659099243/pbdv3jlyd4mhmtis6kqx.jpg';
   const level = 1;
 
+  const wrapper = ({children}) => (
+    <Context.Provider
+      value={{
+        profile: [
+          {isShowHeader: true, myProfile: myProfileMock, navbarTitle: "Who you're following"}
+        ]
+      }}>
+      {children}
+    </Context.Provider>
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `wrapper` function is a good way to provide context for the components in your tests. However, it seems like you're providing an array with a single object as the value of `profile`. If `profile` is expected to be an object, this could lead to issues. Please ensure that the structure of the provided context matches what the components expect.
```diff
-         profile: [
-           {isShowHeader: true, myProfile: myProfileMock, navbarTitle: "Who you're following"}
-         ]
+         profile: {isShowHeader: true, myProfile: myProfileMock, navbarTitle: "Who you're following"}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->